Cargo.toml

```diff
@@ -23,13 +23,18 @@ linux-static-libusb = []
 linux-static-hidraw = []
 linux-shared-libusb = []
 linux-shared-hidraw = []
+linux-shared-udev = ["dep:udev", "dep:nix"]
```

Comment:
hm, probably not a good name for the backend. Maybe rename it to just linux, and make it clear in the lib.rs documentation that the *linux* backend is a native rust implementation (mostly) without C dependencies.